docs/CHANGELOG.asciidoc

```diff
@@ -90,10 +91,356 @@ For detailed information about the n.n.n release, review the following sections.
 
 --
 
+[[release-notes-7.12.0]]
+== {kib} 7.12.0
+
+coming::[7.12.0]
+
+For detailed information about the {kib} 7.12.0 release, review the following information.
+
+[float]
+[[breaking-changes-7.12.0]]
+=== Breaking change
+
+Breaking changes can prevent your application from optimal operation and performance. Before you upgrade to 7.12.0, review the breaking change, then mitigate the
+impact to your application.
+
+// The following section is re-used in the Installation and Upgrade Guide
+
+// tag::notable-breaking-changes[]
```

Comment:
The breaking change didn't get picked up in the Installation and Upgrade guide.  You'll most likely need to make a change in that doc.